gen/testdata/Makefile

```diff
@@ -0,0 +1,18 @@
+ROOT = ../..
+THRIFTRW = $(ROOT)/thriftrw-go
+GO_FILES = $(patsubst %, %.go, $(wildcard *.thrift))
+# We put the generated code in _test files so that they don't get built with the
+# library.
+
+.PHONY: generate
+generate: $(GO_FILES)
+
+.PHONY: clean
+clean:
+	make -C $(ROOT) clean
```

Comment:
I would rather not have `clean` delete anything that is supposed to be checked into the repo.